rstest_macros/src/parse/vlist.rs

```diff
@@ -39,25 +37,51 @@ pub(crate) struct ValueList {
     pub(crate) values: Vec<Value>,
 }
 
+/// Extracts the documentation comment from the attributes.
+fn extract_doc_comment(attrs: &[Attribute]) -> Option<String> {
+    attrs.iter().find_map(|attr| match &attr.meta {
+        Meta::NameValue(MetaNameValue {
+            path,
+            value: Expr::Lit(ExprLit {
+                lit: Lit::Str(doc), ..
+            }),
+            ..
+        }) if path.is_ident("doc") => Some(sanitize_ident(&doc.value().trim(), false)),
+        _ => None,
+    })
+}
+
 impl Parse for ValueList {
     fn parse(input: ParseStream) -> Result<Self> {
         let ident: Ident = input.parse()?;
         let _to: Token![=>] = input.parse()?;
         let content;
         let paren = syn::bracketed!(content in input);
-        let values: Expressions = content.parse()?;
 
-        let ret = Self {
-            arg: ident.into_pat(),
-            values: values.take().into_iter().map(|e| e.into()).collect(),
-        };
-        if ret.values.is_empty() {
+        let mut values = Vec::new();
+        while !content.is_empty() {
+            // Set the description from doc comments if possible
+            let attrs: Vec<Attribute> = content.call(Attribute::parse_outer)?;
+            let description = extract_doc_comment(&attrs);
+
+            // Parse the expression inside the brackets
+            let expr: Expr = content.parse()?;
+            values.push(Value::new(expr, description));
+
```

Comment:
I guess there is a bug here and your test expose it you are ignoring if a comma is present or not : in this case `aaa => 1 2 3` is a legit syntax with the list of 1,2,3